reporters/graphql/umaSubgraph.js

```diff
@@ -0,0 +1,50 @@
+const { GraphQLClient } = require("graphql-request");
+
+/**
+ * Return created graphql client or create a new one.
+ * @return `createdClient` singleton graphql client.
+ */
+let subgraphClient;
+function getUmaClient() {
+  if (!subgraphClient) {
+    subgraphClient = new GraphQLClient("https://api.thegraph.com/subgraphs/name/protofire/uma");
+  }
+  return subgraphClient;
+}
+
+/**
+ * Queries all liquidation related events (LiquidationCreated, LiquidationDisputed, LiquidationWithdrawn)
+ * for a particular EMP
+ * @param {String} empAddress
+ */
+function LIQUIDATION_EVENTS_FOR_EMP(empAddress) {
```

Comment:
this is so useful! next upgrade to bots can be stripping out client logic piece by piece to replace with logic like this. 

great work!